webview/platforms/qt.py

```diff
@@ -726,7 +726,7 @@ def create_submenu(title, line_items, supermenu):
                 m.addSeparator()
             elif isinstance(menu_line_item, MenuAction):
                 new_action = QAction(menu_line_item.title)
-                new_action.triggered.connect(menu_line_item.function)
+                new_action.triggered.connect(Thread(target=menu_line_item.function).start)
```

Comment:
This syntax does not work unfortunately. Nothing happens when menu item is clicked.